docs: Update README.md

tricarro opened the pull request:
<!--────────────────────────────────────────
Checklist before submitting a Pull Request, please ensure you've done the following:
- ✅ Set the "validated" label on this Pull Request if you have the permissions to do so.

- 📝 Write a proper PR title and fill out the description below

- 📖 Read the Contributing guide: 
https://github.com/momentum-design/momentum-design/blob/main/CONTRIBUTING.md

- 🏗️ When making changes to components, follow these conventions: 
https://github.com/momentum-design/momentum-design/tree/main/packages/components/conventions

NOTE: Pull Requests from forked repositories will need to be "validated" by a Momentum Team member before any CI builds will run. Once your PR is "validated", it will be allowed to run subsequent builds without manual approval.
────────────────────────────────────────-->

### Description

Updated the Readme to include an Overview of these assets, Links, Criteria for Inclusion, and Submission guidelines.

tricarro committed:
Update README.md

cscheffauer reviewed (changes requested):
issue: both the PR title and the description are not following our standards.
for the title: please use one of the allowed prefixes, in this case `docs: update README.md` would be applicable.
For the description - it still includes placeholder text for the link. Needs to be removed cause we will get this text in the Release history.

cscheffauer commented on packages/assets/brand-visuals/README.md:
```diff
@@ -2,10 +2,22 @@
+
+Any _**product-specific assets must be independently managed by the respective product teams**_ and will not be included in this centralized library.
+
+## Criteria for Incluseion
```
typo

cscheffauer commented on packages/assets/brand-visuals/README.md:
```diff
@@ -2,10 +2,22 @@
 [![license: mit](https://img.shields.io/badge/License-MIT-blueviolet?style=flat-square)](https://github.com/momentum-design/momentum-design/blob/main/LICENSE) ![State](https://img.shields.io/badge/State-Stable-limegreen?style=flat-square) [![NPM Version](https://img.shields.io/npm/v/%40momentum-design%2Fbrand-visuals)](https://www.npmjs.com/package/@momentum-design/brand-visuals)
 
+## Links
+[🔗 GitHub ](https://github.com/momentum-design/momentum-design/tree/main/packages/assets/brand-visuals)
```
suggestion: i think there is not a huge value having Github and NPM links here, given these connections are clear from the root README.

tricarro committed:
Update README.md

Updated Readme Documentation to include guidelines around assets within the library.

supminn committed:
Merge branch 'main' into tricarro-patch-3
